app/models/animal.rb

```diff
@@ -3,6 +3,13 @@ class Animal < ApplicationRecord
 
   include OrganizationScope
   include CsvExportable
+  include ReportsKit::Model
+
+  reports_kit do
```

Comment:
Could we potentially move this up to the `OrganizationScope` concern, or is it likely not going to be used on many?

Reply:
I tried that. The problem is that a model only accepts a single `reports_kit do` block. I put it here because I anticipate needing to add some more detail here that other models wouldn't share.